grunt/tasks/less/replaceUrls.js

```diff
@@ -0,0 +1,77 @@
+"use strict";
+
+module.exports = function(less) {
+
+    class ReplaceUrls {
+
+        constructor(options) {
+
+            this.isReplacing = true;
+            this.isPreEvalVisitor = true;
+
+            this._options = options;
+            this._visitor = new less.visitors.Visitor(this);
+
+            this._actions = {}
+
+        }
+
+        run(root) {
+            return this._visitor.visit(root);
+        }
+
+        visitRule(ruleNode, visitArgs) {
+            this._inRule = true;
+            return ruleNode;
+        }
+
+        visitRuleOut(ruleNode, visitArgs) {
+            this._inRule = false;
+        }
+
+        visitUrl(URLNode, visitArgs) {
+
+            if (!this._inRule) {
+                return URLNode;
+            }
+            if (URLNode.value && URLNode.value.value && URLNode.value.value.indexOf('#') === 0) {
+              // Might be part of a VML url-node value like:
+              // ``behavior:url(#default#VML);``
+              return URLNode;
+            }
+
+            this._options.replaceUrls.forEach(function(replaceObject) {
+                var matches = URLNode.value.value.match(replaceObject.find);
```

Comment:
I think this should have the same defensive programming as line 37.  Otherwise we'll possibly end up with `"Cannot read property 'match' of undefined"`.

Reply:
Can you give me a console log over URLNode and URLNode.value?